.bandit.yml

```diff
@@ -0,0 +1,4 @@
+# Bandit Config: https://bandit.readthedocs.io/en/latest/config.html
+
+# (optional) list skipped tests here:
+skips: ['B414',]
```

Comment:
I can't find B414 in any list 🙂

Reply:
You are totally right... Let me fix that.